test/runtests.sh

```diff
@@ -164,6 +167,8 @@ fi
 if [ "${RET}" -ne 0 ]; then
 	echo "Tests failed: $FAILED"
 	exit $RET
+elif [ -n "$SKIPPED" ] && [ -n "$TEST_GNU_EXITCODE" ]; then
+	exit 77
```

Comment:
Sorry to dig up an old (merged) PR, but I just came across this hunk and don't really know what it's there for - `TEST_GNU_EXITCODE` doesn't appear to have anything to do with automake tests. @eli-schwartz did you have a specific use for `TEST_GNU_EXITCODE`?

Reply:
IIRC I used that when executing runtests.sh from meson instead of Make, and therefore had a test harness that understood 77.

The idea is that I could then set `TEST_GNU_EXITCODE=1` as an environment for runtests.sh, and then my harness would collect skipped tests and report them as skipped, but the Makefile simply runs it and exits with the exit code of runtests.sh itself and therefore wants skipped tests to exit 0.

Reply:
Ah okay, makes sense I suppose. Just keep in mind that this will also fire for tests which are skipped via `TEST_EXCLUDE` in `config.local`. Thanks for the quick response

Reply:
That should be fine since skipped tests are considered passing, basically.